server/tests/participant.test.ts

```diff
@@ -51,6 +51,10 @@ describe('Participants Endpoint', () => {
     const expectedRes = {
       ...newParticipant,
       preferredLocation: 'Fraser',
+      program: 'HCA',
+      educationalRequirements: 'unknown',
+      driverLicense: 'unknown',
```

Comment:
I've been thinking about how driver license can be unknown, I get this is for current EOI's, but the only values are yes/ no, so true false, would setting this to null be better?